### ISSUE TITLE
Valid post being marked invalid.

### ISSUE THREAD

**Issue by billyp673** [Nano112/quasi-connectivity#34]
I put the post:
https://www.reddit.com/r/minecraft/comments/15oaxvc/im_trying_to_learn_redstone_and_i_want_to_know/
and [quasi-connectivity.com](https://quasi-connectivity.com/) returns "The post seems to be invalid please verify"

**Comment by Nano112** [comment]
Yes currently I am only filtering posts that come from r/redstone, I need to either make the error message clearer or add a clear list of accepted subreddits
